.config/smtp.config.php

```diff
@@ -0,0 +1,15 @@
+<?php
+if (getenv('MAIL_SMTPMODE') == 'smtp') {
+  $CONFIG = array (
+    'mail_smtpmode' => getenv('MAIL_SMTPMODE'),
+    'mail_smtphost' => getenv('MAIL_SMTPHOST'),
+    'mail_smtpport' => getenv('MAIL_SMTPPORT'),
```

Comment:
I think that the if statement should check that every var that does not have a default value is set.
For the most values you can simply use the default values of the docs: https://docs.nextcloud.com/server/12/admin_manual/configuration_server/config_sample_php_parameters.html#mail-parameters

Reply:
Sorry I think I don't quite get what you mean. 
I was thinking to improve it like this:
```php
$CONFIG = array (
  'mail_smtpmode' => getenv('MAIL_SMTPMODE') ?: 'php',
  'mail_smtphost' => getenv('MAIL_SMTPHOST') ?: '127.0.0.1',
  'mail_smtpport' => getenv('MAIL_SMTPPORT') ?: '25',
```
and so on and so on...

Did you refer to that?